src/geo.c

```diff
@@ -805,6 +812,22 @@ void georadiusGeneric(client *c, int srcKeyIndex, int flags) {
         addReplyLongLong(c, returned_items);
     }
     geoArrayFree(ga);
+    return;
+
+src_key_not_exist:
+    if ((flags & GEOSEARCHSTORE) || storekey) {
```

Comment:
In fact, now i think we can delete the `(flags & GEOSEARCHSTORE)`

Reply:
yes, if we're done with the argument parsing, we can just look at `storekey`.